vispy/visuals/filters/markers.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+from vispy.visuals.filters import PrimitivePickingFilter
+
+
+class MarkerPickingFilter(PrimitivePickingFilter):
+    """Filter used to color markers by a picking ID.
+
+    Note that the ID color uses the alpha channel, so this may not be used
+    with blending enabled.
+
+    Examples
+    --------
+    See
+    `examples/scene/marker_picking.py
+    <https://github.com/vispy/vispy/blob/main/examples/scene/marker_picking.py>`_
+    example script.
```

Comment:
We should be able to refer to this with a sphinx gallery reference. I think it is something like:

```
:ref:`sphx-glr-gallery-scene-marker-picking-py`
```

But I could be wrong.

Reply:
Thanks! It took me a bit to get the docs to build locally but this is working:
```
:ref:`sphx_glr_gallery_scene_marker_picking.py`
```